donkeycar/parts/augumentations.py

```diff
@@ -0,0 +1,73 @@
+import cv2
+import numpy as np
+
+
+class Augumentations(object):
```

Comment:
Typo, too many 'u'..., also the filename.

Reply:
British english :smile: